Dockerfile.erb

```diff
@@ -32,6 +32,13 @@ RUN apk add --no-cache coreutils redis
 ADD bin/install-stunnel.sh install-stunnel.sh
 RUN ./install-stunnel.sh
 
+# Use another version of jruby to upgrade jruby-openssl,
+# since the Logstash-packaged version of jruby-openssl can't connect
+# rubygems.org
+RUN apk add jruby && \
```

Comment:
To make sure I understand:

Is the other version of JRuby is installed as part of the Logstash installation package above?

```
RUN curl -O "https://download.elastic.co/logstash/logstash/logstash-${LOGSTASH_VERSION}.tar.gz" && \ # ...
```

And, can we check that the `docker diff` of running this step on top of the previous step does not inadvertently delete any of the Logstash-installed JRuby files?

Assuming so, and assuming this then works, I think this is a clever and good fix. As long as the answer to my second question is yes, this should not affect https://github.com/aptible/gentlemanjerry/pull/38.

Reply:
In trying to build this locally via `TAG=large make build`, I hit:

```
Error: Your application used more memory than the safety cap of 500M.
Specify -J-Xmx####m to increase it (#### = cap size in MB).
Specify -w for full OutOfMemoryError stack trace
```

Is there something extra/different I should to in order to build locally?

Reply:
Diff (and `gem list`) confirms that nothing was deleted. It's 90% additions, with a few changes.

This is probably irrelevant to your error, but I got rid of the tags (managing that has been moved to sweetness), although I do see there's some stuff I missed in the Makefile now.

At which point in the process are you hitting that?

Reply:
>although I do see there's some stuff I missed in the Makefile now.
`.travis.yml`, as well

Reply:
travis.yml updated. That's what the problem was, not the Makefile.